apps/docs/.eslintrc.json

```diff
@@ -26,7 +26,8 @@
         "PreviewComponent": true,
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/d3ce2c08-4b68-4057-a402-9e68000a3d7d)
When clicking on go to next router page in dark mode the success message is not visible

Reply:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/059bbb3b-4b2d-4984-bb3b-7ac1eb188cc6)
There's a todo left.

Reply:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/4e340b3f-c0a8-4e8d-b42d-64db8403265a)
I would remove the notes here.

Reply:
Migration notes were meant as an internal thing for now. I'd be tempted to remove this section from the doc until we formalize the mgiration notes